src/rgw/rgw_rados.h

```diff
@@ -3613,6 +3613,7 @@ class RGWRados : public AdminSocketHook
   int fix_tail_obj_locator(const RGWBucketInfo& bucket_info, rgw_obj_key& key, bool fix, bool *need_fix);
 
   int cls_user_get_header(const string& user_id, cls_user_header *header);
+  int cls_user_clear_header(const string& user_id);
```

Comment:
I prefer cls_user_recalc_stats or cls_user_reset_stats

Reply:
sorry that was an initial method forgot while refactoring, fixing.